app-arch/bzip2/bzip2-1.0.6-r9.ebuild

```diff
@@ -0,0 +1,39 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit flag-o-matic multilib-minimal toolchain-funcs
+
+DESCRIPTION="A high-quality data compressor used extensively by Gentoo Linux"
+HOMEPAGE="http://www.bzip.org/"
+SRC_URI="http://ftp.suse.com/pub/people/sbrabec/bzip2/tarballs/${P}.tar.gz"
```

Comment:
This URL is now wrong. The SUSE tarballs changed their versioning slightly.
I suggest:
```
SUSE_PV="${PV}.0.1"
SUSE_P="${PN}-${SUSE_PV}"
SRC_URI="http://ftp.suse.com/pub/people/sbrabec/bzip2/tarballs/${SUSE_P}.tar.gz"
S="${WORKDIR}/${SUSE_P}"
```

Reply:
Yeah, I was under the thought that gentoo infra would host the patched tarballs without the funky versioning, per my discussion with @SoapGentoo

Reply:
Regardless, the name would HAVE to be different than the original upstream tarballs, otherwise you get different content in the same name. This is good enough for now.

Reply:
Fair enough.

Reply:
@hanetzer prior to mirroring it on Gentoo, let's just use the SUSE URLs; add a block like the above to the ebuild.